provider_azure.go

```diff
@@ -16,10 +16,12 @@ import (
 
 const (
 	// Deprecated: The host of the Azure Active Directory (AAD) graph API
-	azureADGraphHost = "graph.windows.net"
+	azureADGraphHost   = "graph.windows.net"
+	azureADGraphUShost = "https://graph.microsoftazure.us"
```

Comment:
Do you happen to know why the host would include the scheme (`https://`) in the case of `azureADGraphUShost` but not in `azureADGraphHost`?

Reply:
Hello, sorry i did this by mistake in my haste to get a PR up. Should be updated now.

Reply:
Are you able to confirm that you've tested this out and it works as expected? I don't have access to a US Gov L4 environment. This change seems simple enough, so I'm inclined to accept it.

Reply:
@MarkDordoy - Just looking for a response here ^. I will merge this after 👍

Reply:
Hey, I can confirm I tested out the us gov l4 environment response. It's as described in here. I even spoke to MS support about their plan to actually send the correct host but they couldn't provide when.

Reply:
Excellent! Thanks for doing that. I was also wondering when they'd switch the distributed claims hosts over to MS graph.